install.sh

```diff
@@ -16,31 +16,60 @@ CREW_DEST_DIR="${CREW_BREW_DIR}/dest"
 CREW_PACKAGES_PATH="${CREW_LIB_PATH}/packages"
 CURL="${CURL:-curl}"
 
-EARLY_PACKAGES="gcc10 llvm brotli c_ares libcyrussasl libiconv libidn2 \
-libmetalink libnghttp2 libpsl libssh2 libtirpc libunistring openldap \
-rtmpdump zstd ncurses ca_certificates ruby libffi openssl nettle krb5 \
-p11kit libtasn1 gnutls curl git icu4c libedit"
+EARLY_PACKAGES=(gcc10 libiconv libssh2 ncurses ruby openssl git)
```

Comment:
Didn't we need to add curl because i686 Chromebooks were misbehaving during install?

Reply:
IIRC some i686 images have a broken curl in them... which is a big reason NOT to do this at all.

Reply:
```bash
case $(uname -m) in ARCH
   i*86)
    early-packages+="curl"
    ;;
esac
```

Or however that would be in Ruby, maybe.

Reply:
That could work.  I'm all for reducing the size and scope of core packages.